src/gems/model/parsing.py

```diff
@@ -87,10 +96,28 @@ class InputModel(ModifiedBaseModel):
     port_field_definitions: List[InputPortFieldDefinition] = Field(default_factory=list)
     binding_constraints: List[InputConstraint] = Field(default_factory=list)
     constraints: List[InputConstraint] = Field(default_factory=list)
+
+    # --- new field ---
+    objective_contributions: Optional[List[InputObjectiveContribution]] = Field(
+        default_factory=lambda: [], alias="objective-contributions"
```

Comment:
```suggestion
        default_factory=list, alias="objective-contributions"
```
I think this does the same, but more consistent with previous code